src/bitmessagekivy/tests/test_create_random_address.py

```diff
@@ -63,13 +65,11 @@ def test_set_default_address(self):
         # This is for opening side navbar
         self.open_side_navbar()
         # Click to open Address Dropdown
-        self.cli.wait_click('//NavigationItem[0]/CustomSpinner[0]', timeout=5)
+        self.cli.wait_click('//NavigationItem[0]/IdentitySpinner[0]', timeout=5)
```

Comment:
- better `NavigationItem` as there are `NavigationItem`s at two different levels
- reference `IdentitySpinner` by key, not by index, e.g. `IdentitySpinner[@id=\"identity_dropdown\"]`